flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/sink/ElasticsearchWriterITCase.java

```diff
@@ -496,6 +496,19 @@ public void execute(
             }
         }
 
+        @Override
+        public void execute(
```

Comment:
@vahmed-hamdy you may apply the same change as we did for OpenSearch [1], so to be complaint with 1.20/1.19/1.18/1.17.

[1] https://github.com/apache/flink-connector-opensearch/pull/48

Reply:
@reta thanks for the hint, PR updated